olive/strategy/utils.py

```diff
@@ -65,8 +65,8 @@ def _search_space_graph(search_space: Dict[str, SearchParameter]) -> DirectedGra
     """Create a directed graph from the search space."""
     graph = DirectedGraph(list(search_space.keys()))
     for name, param in search_space.items():
-        if isinstance(param, Conditional):
-            for parent in param.parents:
+        if isinstance(param, SearchParameter):
```

Comment:
this condition is unnecessary since all params in the search space must be search parameters